shared/utils/build-all-guides.py

```diff
@@ -182,42 +182,54 @@ def get_cpu_count():
 
 
 def main():
-    usage = "usage: %prog [options]"
-    parser = OptionParser(usage=usage)
-    parser.add_option(
-        "-i", "--input", dest="input_content", type="string",
-        action="store", help="INPUT can be XCCDF or Source DataStream. XCCDF "
-        "is supported with all OpenSCAP versions. You need OpenSCAP 1.1.0 or "
-        "higher to generate guides from Source DataStream!"
-    )
-    parser.add_option(
-        "-j", "--jobs", dest="parallel_jobs", type="int",
-        action="store", help="How many workers should generate guides in "
-        "parallel. Defaults to the number of available CPUs.",
-        default=get_cpu_count()
-    )
-    (options, args) = parser.parse_args()
+    p = argparse.ArgumentParser()
+
+    sp = p.add_subparsers(dest="cmd", help="actions")
+
+    make_sp = sp.add_parser("build", help="Build all the HTML guides")
+    make_sp.set_defaults(cmd="build")
+
+    input_sp = sp.add_parser("list-inputs", help="Generate input list")
+    input_sp.set_defaults(cmd="list_inputs")
```

Comment:
Can you explain why that would happen? list-inputs is a subparser and nothing else, it tells argparse which subparser to apply. One argparse applies the subparser it sets the "cmd" attribute of args to "list_inputs" as instructed by the "set_defaults" line.